client/src/components/pages/Profile/ProfilePic/style.css

```diff
@@ -0,0 +1,28 @@
+.profile-pic {
+    width: 100px;
+    height: 150px;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    align-items: center;
+}
+
+.profile-pic--img {
+    width: 100%;
+    height: 100px;
+    border: 2px solid var(--primary-color, pink);
+    border-radius: 50%;
```

Comment:
@ahmedisam99 You could try out the css `object-fit` property on this img, which might look nice.  Then if your image element's aspect ratio (which I think your css is making it a 100px square) doesn't match up with the actual image file's aspect ratio, the browser will scale and center the image for you.

For example, you could use `object-fit: cover` to have the browser scale up and crop the image for you.

https://developer.mozilla.org/en-US/docs/Web/CSS/object-fit

Reply:
you mean instead of explicitly defining the width and height for the image?

Reply:
You would do both: set the img element’s width / height and also set the img object-fit. That will let you control what happens when the img element and the actual image file don’t have the same aspect ratio (without object-fit, the image could appear squashed or stretched)